### DISCUSSION TITLE
10.8.0-1

### DISCUSSION THREAD

**Discussion by pdulvp** [pdulvp/jellyfin-qnap#21] [Announcements]
The Jellyfin 10.8.0 has been released.
Do you manage to make ffmpeg work properly with your QNAP ?

**Comment by nzapodege** [comment]
I can not, the cpu with many movies goes full. I am still using the jellyfin_10.8.0-b1.qpkg. Mine is a Qnap TS-253-d

**Reply by pdulvp** [reply]
mine TS-253-A is lagging. it plays 2sec then wait a bit, then plays 2sec..

**Reply by nzapodege** [reply]
My 253-d plays the audio correctly

**Comment by cyril-cheng** [comment]
I can not, the cpu with many movies goes full. I am using the jellyfin_10.8.0-1.qpkg.Mine is a TS-453Dmini.

**Reply by cyril-cheng** [reply]
QTS 5.0.1.2034

**Comment by menstry** [comment]
QTS 5.0.1.2034 can not cpu:G5400  mine is tvs-672x for patched

**Comment by thofei** [comment]
Hi,

installed it on my TS 431-P3 (Annapurna Alpine AL314 processor), but when i try to open it, it just opens a new website saying ERR_CONNECTION_REFUSED (Edge Browser).

Am i missing something? I'm totally new to QNAP's (or NAS in general, so bear with me).

Thanks
T.

**Comment by nzapodege** [comment]
even with this version with ffmpeg 4.4.1-4, some movies do not start and the cpu goes to full, forcing me to restart the qnap
